infobar-with-cta/template.ts

```diff
@@ -12,16 +12,13 @@ export class InfobarWithCTATemplate implements CampaignTemplateComponent {
     ])
     style: StyleField = { label: "Light on Dark", className: "evg-light-on-dark" };
 
+    @subtitle("Optional text field")
     messageText: string = "Infobar Message Text";
 
-    messageVisibility: boolean = true;
-
     @title('CTA Text')
+    @subtitle("Optional text field")
     ctaText: string = "Call To Action";
 
-    @title('CTA Visibility')
-    ctaVisibility: boolean = true;
-
     @title('CTA Destination URL')
```

Comment:
Could you make this minor update too?
```suggestion
    @title("CTA Destination URL")
```